apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,438 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable prettier/prettier */
+/* eslint-disable */
+
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Center,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  List,
+  ListItem,
+  Text,
+  VStack,
+  useToken,
+} from '@chakra-ui/react'
+import { Button, BxRightArrowAlt } from '@opengovsg/design-system-react'
+import Preview from '../editing-experience/components/Preview'
+
+import { Icon } from '@chakra-ui/react'
+import { BiLeftArrowAlt, BiShow } from 'react-icons/bi'
+
+interface LayoutSelectionProps {
+  pageName: String
```

Comment:
i think you can introduce types for the different types of layouts we are supporting, can see on figma

Reply:
Not sure if this was what you meant, but let me know if you meant something else!